tests/test_cache.py

```diff
@@ -57,6 +57,19 @@ def test_get_cache_key(self):
         key = cache_backend.get_cache_key(SampleResourceModel, url)
         assert key == "note::http://foo.com/v1/expected_title/"
 
```

Comment:
This is no longer about cache keys, I think the test name test_get_cache_key_with_parameters is misleading